src/intent/planner.py

```
@@ -0,0 +1,51 @@
+# planner.py
+
+from typing import List
+from intent.detector import Intent
+
+class InstallationPlanner:
+
+    GPU_PACKAGES = ["cuda", "cudnn", "pytorch", "tensorflow"]
```

Ruff (0.14.6)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)